src/lib/certmap/sss_cert_content_crypto.c

```diff
@@ -662,10 +713,11 @@ static int get_san(TALLOC_CTX *mem_ctx, X509 *cert, struct san_list **san_list)
                 goto done;
             }
 
-            /* i2d_TYPE increment the second argument so that it points to the end of
-             * the written data hence we cannot use i->bin_val directly. */
+            /* i2d_EDIPARTYNAME increment the second argument so that it points
+             * to the end of the written data hence we cannot use data directly.
+             */
             p = data;
-            len = i2d_EDIPARTYNAME(current->d.ediPartyName, &data);
+            len = i2d_EDIPARTYNAME(current->d.ediPartyName, &p);
```

Comment:
Wouldn't it be better to split this in a separate patch?

Reply:
Done.